pkg/remotewrite/config.go

```diff
@@ -21,6 +21,8 @@ const (
 	defaultMetricPrefix = "k6_"
 )
 
+var defaultTrendStats = []string{"p(99)"}
+
```

Comment:
@na-- @oleiade it would be good to get an opinion on this. I took this decision autonomously, but you could have a different opinion.

Reply:
For me it makes some sense that the default value should match whatever the value of `--summary-trend-stats` in k6 is (default or user-supplied). But I can also see why that might be a bit too expensive to maintain by default... So, I don't have a strong opinion either way, sorry.